src/api/submission/parser.py

```diff
@@ -14,6 +14,12 @@
     "get_site_id",
 )
 
+def _cast_decimal_to_str(val):
+    if val is None:
+        return
```

Comment:
Should this be `return None`?
Should we catch decimal conversion exceptions?

Reply:
It does return None, but I've changed it to explicitly return `None`.  I'm not sure if there's anything to capture since it's converting to string.  Also, it would be captured further upstream in a try/except